docs/io/unified.rst

```diff
@@ -318,6 +316,15 @@ the keywords ``DATE``, ``DATE-*`` (ISO-8601 datetime strings) and the ``MJD-*``
 For more details regarding the FITS time paper and the implementation,
 refer to :ref:`fits_time_column`.
 
+As opposed to the default write behaviour of a `~astropy.time.Time` mixin column
```

Comment:
I'd again rephrase the rationale (linking directly to the Time documentation):
```
Since not all FITS readers are able to use the FITS time standard, it is also
possible to store `~astropy.time.Time` instances using the `_time_format`.
For this case, none of the ...
```